addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -702,3 +480,184 @@ a.no-decoration {
 .o_shadow_animation {
     box-shadow: 0 0 5px 10px rgba(240,8,0,0.4)!important;
 }
+
+#wrapwrap {
+    background-color: $gray-200;
+}
```

Comment:
@qha-odoo You just can't style the whole website and ignore others apps just because the eCommerce is installed. The color of the body is the one chosen by the user and this prevents the user to choose one. If we want to change the default color of the body, this must be a website feature to discuss. If we want to allow the user to choose a different body color just for the eCommerce, this is a whole new feature too. I think we should be happy to leave the body color white for now.